sql/views.sql

```diff
@@ -326,6 +327,11 @@ SELECT
     END
     AS err_message
 FROM
-    _timescaledb_internal.job_errors;
-
+    _timescaledb_internal.job_errors
+LEFT JOIN
+    _timescaledb_config.bgw_job ON (bgw_job.id = job_errors.job_id)
+WHERE
+    pg_catalog.pg_has_role(current_user, 'pg_database_owner', 'MEMBER')
```

Comment:
Tricky thing is that `pg_database_owner` is only available in PostgreSQL 14 and higher.

Reply:
Yup, used a sub-select instead to pick the database owner. We can simplify this once we are deprecating PG13 or before, if we introduce a similar role in our own code.